Extensions/xkit_updates.js

```diff
@@ -292,7 +292,7 @@ XKit.extensions.xkit_updates = new Object({
 				try {
 					var mdata = JSON.parse(response.responseText);
 
-					if (mdata.malicious === true || mdata.malicious == "true") {
+					if (mdata.malicious || mdata.malicious == "true") {
```

Comment:
The original intent here seems to be "if `mdata.malicious` is true (either as a boolean or a string)", so here I would prefer to keep the original line.